Postman/Postman.js

```diff
@@ -107,111 +101,108 @@ Seriesコレクションから新刊のBookListドキュメントを抽出する
+  })
+  .fail((err)=> {
+    log.info(err);
+  });
 };
 
-
-
-Postman.prototype.sentSeries = function(user){
-	var args = Array.prototype.slice.call(arguments);
-			args = _.compact(args);
-	var done = args[args.length - 1];
-
-	var _Mailer = this.Mailer;
-
-	var _fetchSeries = this.Utils.defer(this.fetchSeries.bind(this));
-	var _inspectSeries = this.Utils.defer(this.inspectSeries.bind(this));
-
-	var _createTemplate = this.Utils.defer(function(series, done){
-		_Mailer.createTemplate('series', series, function(err, mailStrings){
-			if(err){
-				return done(err);
-			}
-			_Mailer.setMail("info@kindlized.it", user.mail, "[kindlize.it] 新刊通知", mailStrings);
-			done();
-		});
-	});
-
-	var _sendMail = this.Utils.defer(_Mailer.send.bind(_Mailer));
-
-	Q.when(user)
-	.then(_fetchSeries)
-	.then(_inspectSeries)
-	.then(_createTemplate)
-	.then(_sendMail)
-	.then(function(series){
-		done(null, series);
-	})
-	.fail(function(err){
-		done(null, err);
-	});
+Postman.prototype.sentSeries = function sentSeries(user) {
+  let args = Array.prototype.slice.call(arguments);
+  args = _.compact(args);
+
+  const done = args[args.length - 1];
+  const _Mailer = this.Mailer;
+
+  const _fetchSeries = this.Utils.defer(this.fetchSeries.bind(this));
+  const _inspectSeries = this.Utils.defer(this.inspectSeries.bind(this));
+
+  const _createTemplate = this.Utils.defer((series, callback)=> {
+    _Mailer.createTemplate('series', series, (err, mailStrings)=> {
+      if (err) {
+        return callback(err);
+      }
+      _Mailer.setMail('info@kindlized.it', user.mail, '[kindlize.it] 新刊通知', mailStrings);
+      callback();
+    });
+  });
+
+  const _sendMail = this.Utils.defer(_Mailer.send.bind(_Mailer));
+  Q.when(user)
+  .then(_fetchSeries)
+  .then(_inspectSeries)
+  .then(_createTemplate)
+  .then(_sendMail)
+  .then((info)=> {
+    return done(null, info);
+  })
+  .fail((err)=> {
+    return done(null, err);
+  });
 };
 
 
-Postman.prototype.runSeries = function(){
-	var _fetch = this._defer(this.fetch.bind(this));
-	var _sentSeries = this.Utils.defer(this.sentSeries.bind(this));
-	var _sentAllUsers = this.Utils.defer(function(users, done){
-
-		Q.all(users.map(_sentSeries))
-		.then(function(sendStatus){
-			done(null, sendStatus);
-		})
-		.fail(function(err){
-			done(err);
-		});
-	});
-
-	Q.when()
-	.then(_fetch)
-	.then(_sentAllUsers)
-	.then(function(sendStatus){
-		return log.info(sendStatus);
-	})
-	.fail(function(err){
-		return log.info(err);
-	});
+Postman.prototype.runSeries = function runSeries() {
+  const _fetch = this._defer(this.fetch.bind(this));
+  const _sentSeries = this.Utils.defer(this.sentSeries.bind(this));
+  const _sentAllUsers = this.Utils.defer((users, done)=> {
+    Q.all(users.map(_sentSeries))
+    .then((sendStatus)=> {
+      return done(null, sendStatus);
+    })
+    .fail((err)=> {
+      return done(err);
+    });
+  });
+
+  Q.when()
+  .then(_fetch)
+  .then(_sentAllUsers)
+  .then((sendStatus)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).